src/app/components/media/CreateMediaInput.js

```diff
@@ -109,13 +120,17 @@ class CreateMediaInput extends React.Component {
     if (video !== '') {
       title = video.name;
     }
+    if (audio !== '') {
+      title = audio.name;
+    }
 
-    if (url || quote || image || video) {
+    if (url || quote || image || video || audio) {
       return ({
         url,
         quote,
         image,
         video,
+        audio,
```

Comment:
can we simplify this and merge all these different media type variables into, say, file ?

Reply:
Done